### PR TITLE
fix: fix default value cannot accept negative number

### LINKED ISSUE #3204: Failed to create table with negative default value
### What type of bug is this?

Unexpected error

### What subsystems are affected?

Other

### Minimal reproduce step

```sql
CREATE TABLE quidem(
  consequatur BIGINT DEFAULT 5664459986523542528,
  alias SMALLINT NOT NULL,
  velit FLOAT,
  repellendus SMALLINT,
  quod BIGINT DEFAULT -6372301239504951450,
  nihil TIMESTAMP(9) TIME INDEX
) ENGINE = mito;
```

### What did you expect to see?

It should be able to create the table.

### What did you see instead?

message: “Unsupported expr in default constraint: UnaryOp { op: Minus, expr: Value(Number(\“6372301239504951450\“, false)) } for column: quod”

### What operating system did you use?

not matter

### What version of GreptimeDB did you use?

0.6.0

### Relevant log output and stack trace

_No response_

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

fix default value cannot accept negative number

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

## Refer to a related PR or issue link (optional)
close #3204

### CODE REVIEW

**Review by v0y4g3r — APPROVED** [review]
LGTM

File: tests/cases/standalone/common/insert/insert.sql [hunk 1]
```diff
@@ -23,3 +23,12 @@ insert into presentations values (1, 'Patrick Damme', 'Analytical Query Processi
 DROP TABLE integers;
 
 DROP TABLE presentations;
+
+ CREATE TABLE neg_default_value(i INT DEFAULT -1024, ts TIMESTAMP TIME INDEX);
```
Comment (QuenKar) [inline tests/cases/standalone/common/insert/insert.sql:27]: Is there an extra space here?

Comment (WenyXu) [reply tests/cases/standalone/common/insert/insert.sql:27]: Yes, I'm going to remove it.

**Review by QuenKar — APPROVED** [review]
LGTM

File: tests/cases/standalone/common/insert/insert.sql [hunk 2]
```diff
@@ -23,3 +23,12 @@ insert into presentations values (1, 'Patrick Damme', 'Analytical Query Processi
...
+
+ INSERT INTO neg_default_value(ts) values ('2000-01-01 00:00:00+00:00');
+
+ SELECT * FROM neg_default_value;
+
+ DROP TABLE neg_default_value;
+
```
Comment (killme2008) [inline tests/cases/standalone/common/insert/insert.sql:34]: Needs a blank line at the end

**Review by killme2008 — APPROVED** [review]

**Pull request merged**